components/Cards/src/MDCCollectionViewCardCell.m

```diff
@@ -0,0 +1,175 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCCollectionViewCardCell.h"
+
+
+@interface MDCCollectionViewCardCell ()
+
+@property(nonatomic, assign) CGPoint lastTouch;
+
+@end
+
+@implementation MDCCollectionViewCardCell
+
+- (instancetype)initWithCoder:(NSCoder *)coder {
+  self = [super initWithCoder:coder];
+  if (self) {
+    [self commonInit];
+  }
+  return self;
+}
+
+- (instancetype)initWithFrame:(CGRect)frame {
+  self = [super initWithFrame:frame];
+  if (self) {
+    [self commonInit];
+  }
+  return self;
+}
+
+- (void)commonInit {
+  self.cardView = [[MDCCard alloc] initWithFrame:self.contentView.bounds
+                   withIsUsingCollectionViewCell:YES];
+  self.cardView.autoresizingMask =
+    UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+  self.contentView.autoresizingMask =
+    UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+  [self.contentView addSubview:self.cardView];
+  self.cornerRadius = 4.f;
+  self.editMode = NO;
+}
+
+- (void)layoutSubviews {
+  self.shadowElevation = 1.f;
```

Comment:
I would typically expect to set shadowElevation in an initializer or during a state change.
Can we move this there and get rid of this  method completely?